types/preview/index.d.ts

```diff
@@ -514,11 +533,12 @@ export type WorkbenchBackendActionForInsertDetail = {
   sort?: number;
   portal?: boolean;
   bg?: boolean;
-  nodeData: BuilderRuntimeNode;
+  nodeData: BuilderRuntimeNode & SnippetRuntimeNode;
   dragOverInstanceId?: string;
   dragStatus?: dragStatus;
   sortData?: WorkbenchSortData;
   type: "brick" | "provider";
+  runtimeContext?: SnippetRuntimeContext;
```

Comment:
叫 `snippetContext` 吧

```suggestion
  snippetContext?: SnippetRuntimeContext;
```